src/Commands/UserCommands/CancelCommand.php

```diff
@@ -54,15 +60,24 @@ public function execute()
      */
     public function executeNoDB()
     {
-        //Database not setted or without connection
-        $message = $this->getMessage();
-        $chat_id = $message->getChat()->getId();
+        //Output the default message when there is no DB connection
+        parent::executeNoDB();
```

Comment:
You don't :smiley: 

Feel free to change this and output any message.
I just set it to what other commands output when there is no DB connection.

You're right, it makes more sense to just output "No active conversation!" or something :+1: